pkgs/servers/tailscale/default.nix

```diff
@@ -2,23 +2,23 @@
 
 buildGoModule rec {
   pname = "tailscale";
-  version = "0.97-219";
+  version = "0.98-84";
```

Comment:
This package should probably follow Tailscale "stable" releases, not the "often broken" unstable track. The stable version is listed at the bottom of https://pkgs.tailscale.com/stable . This should be `0.98-0`, with a checkout rev of 9467acda8407490ce57d553ddc733dca728313dc.